### PR TITLE
sysdig doc-as-code blog

### PR DESCRIPTION
**Pull request by radhikapc**
Signed-off-by: radhikapc <radhika.dwaraka@gmail.com>

### CODE REVIEW

File: _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Adopting Docs-as-Code: From Hackathon to Production"
+excerpt: Sysdig's journey from unstructured to structured, finally to semi-structured authoring and how Sysdig hackathon enabled design their homegrown docs-like-code solution.
+last_modified_at: 
+categories: articles
```
Comment (annegentle) [inline _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md:6]: Let's try the `canonical_url: "https://sysdig.com/blog/adopting-docs-as-code/"` line here in the front matter and see if the Jekyll SEO plugin works like I think it should.

File: _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Adopting Docs-as-Code: From Hackathon to Production"
+excerpt: Sysdig's journey from unstructured to structured, finally to semi-structured authoring and how Sysdig hackathon enabled design their homegrown docs-like-code solution.
+last_modified_at:
```
Comment (annegentle) [inline _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md:5]: You can enter `last_modified_at: Mon Jul 11 17:33:53 CDT 2022` to match the format of other posts.

File: _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+| ----------------------------------------- | ------------------------------------------------------------ |
+| Tighter collaboration with SMEs           | We are still exploring and experimenting and will be back with our learnings! |
+| Easy integration with Sysdig repositories |                                                              |
+| Easier versioning branching for releases  |                                                              |
+| Staging / Preview builds                  |                                                              |
+| Enhanced search                           |                                                              |
+| Dark mode                                 |                                                              |
+| RSS feed                                  |                                                              |
+| In-house doc infrastructure               |                                                              |
+
```
Comment (annegentle) [inline _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md:103]: These tables look great! They really drew me to the post.

File: _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+The result was a Python script to solve this problem and make the conversion automated.
+
+### CI/CD Pipeline
+
+We chose [Hugo](https://gohugo.io/) static site generator with [Docsy](https://www.docsy.dev/) theme for building the documentation website for its rich feature set. Hugo is faster, offers a rich feature set for expansion, and has an active community for support. Docsy offers features such as dark theme, RSS feed support, and swagger integration for APIs, which we plan to introduce to the site at some point. [Netlify](https://www.netlify.com/) is faster to launch and easier to integrate with Github repository for hosting preview and production builds. The publishing flow has been designed in such a way that each Pull Request would generate a preview build so reviewers can see the content as it would appear on the website. This mechanism accelerated our documentation publishing pace without sacrificing quality.
+
```
Comment (annegentle) [inline _posts/articles/2022-07-11-sysdig-adopting-docs-as-code.md:78]: There's an image on the blog that's missing here; can you add it? It's https://sysdig.com/wp-content/uploads/adopting-doc-as-code-01.png

**Review by annegentle — COMMENTED** [review]
Thanks for this! A couple of inline comments and then it should be good-to-go.

**Comment by radhikapc** [comment]
Incorporated the changes. Thanks again, @annegentle

**Review by annegentle — APPROVED** [review]
Looks good! I'll merge and then send a new Pull Request to merge the `build` branch to `main`, which publishes the post. Thanks so much!

**Comment by radhikapc** [comment]
> Looks good! I'll merge and then send a new Pull Request to merge the `build` branch to `main`, which publishes the post. Thanks so much!

Thank you

**Pull request merged**